firestore/tests/system.py

```diff
@@ -768,7 +768,12 @@ def test_get_all(client, cleanup):
 
     assert snapshots[0].exists
     assert snapshots[1].exists
+
+    # missing document attribute
     assert not snapshots[2].exists
+    assert snapshots[2].id == "b"
+    assert not snapshots[2]._data
```

Comment:
Please move these assertions to the unit test below/

Reply:
Done